apps/heureka/src/components/filters/FilterSelect.jsx

```diff
@@ -65,7 +73,7 @@ const FilterSelect = ({ entityName, isLoading, filterLabels, filterLabelValues,
         <Button label="Clear all" onClick={() => clearActiveFilters(entityName)} variant="subdued" />
       )}
       <SearchInput
-        placeholder="Search term"
+        placeholder={`search term for ${searchTargets}`}
```

Comment:
I think that could also work:

placeholder=\`search term for ${searchTargets}\`

Did you try it?